### ISSUE TITLE
thinThickBarlineSeparation not listed in engraving defaults

### ISSUE THREAD

**Issue by craigvear** [steinbergmedia/bravura#69]
I've noticed thinThickBarlineSeparation is not listed in the engraving defaults. Is this normal?

**Comment by rettinghaus** [comment]
Should the space between two thin barlines be different?

**Comment by craigvear** [comment]
Looking at the engraving defaults in SMuFL there are several listed for barlines, and not all are in Bravura:


"thinBarlineThickness" | The thickness of a thin barline, e.g. a normal barline, or each of the lines of a double barline
-- | --
"thickBarlineThickness" | The thickness of a thick barline, e.g. in a final barline or a repeat barline
"dashedBarlineThickness" | The thickness of a dashed barline
"dashedBarlineDashLength" | The length of the dashes to be used in a dashed barline
"dashedBarlineGapLength" | The length of the gap between dashes in a dashed barline
"barlineSeparation" | The default distance between multiple thin barlines when locked together, e.g. between two thin barlines making a double barline, measured from the right-hand edge of the left barline to the left-hand edge of the right barline.
"thinThickBarlineSeparation" | The default distance between a pair of thin and thick barlines when locked together, e.g. between the thin and thick barlines making a final barline, or between the thick and thin barlines making a start repeat barline.
"repeatBarlineDotSeparation" | The default horizontal distance between the dots and the inner barline of a repeat barline, measured from the edge of the dots to the edge of the barline.

**Comment by rettinghaus** [comment]
Ok, thanks for the clarification. I overlooked thinThickBarlineSeparation.

**Comment by alanwhite** [comment]
Could we clarify if Bravura will be adding the thinThickBarlineSeparation as required by the SMuFL spec plse?

**Comment by dspreadbury** [comment]
Yes, when I next review the metadata for Bravura, I'll take care of this.
